dotCMS/src/main/java/com/dotcms/rest/api/v1/experiments/ExperimentsResource.java

```diff
@@ -500,6 +518,45 @@ public ResponseEntityExperimentResults getResult(@Context final HttpServletReque
         return new ResponseEntityExperimentResults(experimentResults);
     }
 
+    /**
+     * Healthcheck for the Experiments/Analytics configuration.
+     *
+     */
+    @GET
+    @NoCache
+    @Path("/health")
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public ResponseEntityView healthcheck(@Context final HttpServletRequest request,
+            @Context final HttpServletResponse response)
+            throws DotDataException, DotSecurityException, SystemException, PortalException {
+
+        final InitDataObject initData = getInitData(request, response);
+        final Host host = WebAPILocator.getHostWebAPI().getCurrentHost(request);
+        final AnalyticsApp analyticsApp = Try.of(()->AnalyticsHelper.get().appFromHost(host))
+                .getOrNull();
+
+        if(analyticsApp==null) {
+            return new ResponseEntityView<>(Map.of("healthy", false));
+        }
+
+        final String url = analyticsApp.getAnalyticsProperties().analyticsWriteUrl();
+
+        final CircuitBreakerUrlBuilder builder = CircuitBreakerUrl.builder()
+                .setMethod(Method.POST)
+                .setUrl(url)
+                .setParams(map("token", analyticsApp.getAnalyticsProperties().analyticsKey()))
+                .setTimeout(4000)
+                .setHeaders(POSTING_HEADERS);
```

Comment:
I think we should creare a Util for this, also we should check that it connect to CubeJs too.


Also we should check that this is not set analyticsApp.getAnalyticsProperties().analyticsKey() before hit the Jitsu or CubeJs End point